src/dev/code_coverage/ingest_coverage/integration_tests/team_assignment.test.js

```diff
@@ -38,11 +37,14 @@ describe('Team Assignment', () => {
   describe(`when the codeowners file contains #CC#`, () => {
     it(`should strip the prefix and still drill down through the fs`, async () => {
       const { stdout } = await execa('grep', ['tre', teamAssignmentsPath], { cwd: ROOT_DIR });
-      expect(stdout).to.be(`x-pack/plugins/code/jest.config.js kibana-tre
-x-pack/plugins/code/server/config.ts kibana-tre
-x-pack/plugins/code/server/index.ts kibana-tre
-x-pack/plugins/code/server/plugin.test.ts kibana-tre
-x-pack/plugins/code/server/plugin.ts kibana-tre`);
+      const lines = stdout.split('\n').filter((line) => !line.includes('/target'));
+      expect(lines).toEqual([
+        'x-pack/plugins/code/jest.config.js kibana-tre',
+        'x-pack/plugins/code/server/config.ts kibana-tre',
+        'x-pack/plugins/code/server/index.ts kibana-tre',
+        'x-pack/plugins/code/server/plugin.test.ts kibana-tre',
+        'x-pack/plugins/code/server/plugin.ts kibana-tre',
+      ]);
```

Comment:
Moving the plugin to its own project created a `target` folder to be added, but this script is not ignoring them, so had to adapt the test